### PR TITLE
-Added Go Bootcamp and Python App deploy

### PR DESCRIPTION
**Pull request by fcpimenta**
## What does this PR do?
-Add Go Bootcamp by Jefferson Otoni Lima - Go Community Leader
-Add a course about the deployment workflow using Python as a base language by Cássio Botaro. Also add a course about API's using Python - Python Community Leader

### Why is this valuable (or not)?

Jefferson and Cássio are very active leaders in the Brazilian community and help many people to learn and develop.

### How do we know it's really free?
https://github.com/jeffotoni/goworkshopdevops/blob/master/LICENSE
https://github.com/cassiobotaro/do_zero_a_implantacao/blob/master/LICENSE
https://github.com/luizalabs/tutorial-python-brasil/blob/main/LICENSE

### For book lists, is it a book? For course lists, is it a course? etc.
They are courses/bootcamps

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -246,12 +246,14 @@
 
 * [Algoritmos em Python](https://algoritmosempython.com.br) - Douglas do Couto
 * [Aprenda Python 3 em 6 horas](https://www.udemy.com/course/aprenda-python-3-em-6h/) - Alcimar A. Costa (Udemy)
+* [Construindo API's robustas utilizando Python](https://github.com/luizalabs/tutorial-python-brasil) - Cássio Botaro
```
Comment (LuigiImVector) [inline courses/free-courses-pt_BR.md:249]:
```suggestion
* [Construindo API's robustas utilizando Python](https://github.com/luizalabs/tutorial-python-brasil) - Cássio Botaro et al.
```

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -246,12 +246,14 @@
...
 * [Curso de Programação em Python](https://www.youtube.com/playlist?list=PLFKhhNd35zq_INvuX9YzXIbtpo_LGDzYK) - Prime Cursos do Brasil
 * [Curso de Python](https://www.youtube.com/playlist?list=PLesCEcYj003QxPQ4vTXkt22-E11aQvoVj) - eXcript (YouTube)
 * [Curso de Python 3 - Mundo 1: Fundamentos](https://www.youtube.com/playlist?list=PLHz_AreHm4dlKP6QQCekuIPky1CiwmdI6) - Gustavo Guanabara (Curso em Vídeo)
 * [Curso em vídeo - Python](https://www.youtube.com/playlist?list=PLvE-ZAFRgX8hnECDn1v9HNTI71veL3oW0) - Joao Pedro Araujo
 * [Data Science: Visualização de Dados com Python](https://www.udemy.com/visualizacao-de-dados-com-python/) - Diego Mariano (Udemy)
 * [Django 2.0 - Aprendendo os conceitos fundamentais](https://www.udemy.com/django-20-aprendendo-os-conceitos-fundamentais/) - Gregory Pacheco (Udemy)
+* [Do zero à implantação utilizando Python](https://github.com/cassiobotaro/do_zero_a_implantacao) - Cássio Botaro
```
Comment (LuigiImVector) [inline courses/free-courses-pt_BR.md:256]:
```suggestion
* [Do zero à implantação utilizando Python](https://github.com/cassiobotaro/do_zero_a_implantacao) - Cássio Botaro et al.
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -388,6 +388,7 @@
 ### Go
 
 * [A Tour Of Go](https://tour.golang.org/welcome/1)
+* [DevOps BootCamp](https://github.com/jeffotoni/goworkshopdevops) - Jefferson Otoni Lima
```
Comment (LuigiImVector) [inline courses/free-courses-en.md:391]:
```suggestion
* [DevOps BootCamp](https://github.com/jeffotoni/goworkshopdevops) - Jefferson Otoni Lima et al.
```

**Review by LuigiImVector — COMMENTED** [review]
I have added clarifications regarding authors

**Comment by eshellman** [comment]
It's ok to leave off the "et. al." when there are minor contributors. Consider the suggestions, we can merge with or without.

**Comment by LuigiImVector** [comment]
thanks for the clarification, I'm learning every day on how to best review PRs

**Pull request merged**